### ISSUE TITLE
Noisy tabset is not easily selectable when window is maximised

### ISSUE THREAD

**Issue by srirambv** [brave/browser-laptop#8764]
- Did you search for similar issues before submitting this one?
Yes

- Describe the issue you encountered:
Noisy tabset is not easily selectable when window is maximised

- Platform (Win7, 8, 10? macOS? Linux distro?):
Windows 10 x64

- Brave Version (revision SHA):
Brave	0.15.3
rev	1c04c53

- Steps to reproduce:
    1. Have enough tabs to create tabsets
    2. Play audio/video in one of the tab
    3. Switch tabsets, try switching back to the noisy tabset, very hard to switch 
    
- Actual result:
Noisy tabset is not easily selectable when window is maximised

- Expected result:
Should be easy to select noisy tabset

- Will the steps above reproduce in a fresh profile? If not what other info can be added?
Yes

- Is this an issue in the currently released version?
No

- Can this issue be consistently reproduced?
Yes

- Extra QA steps:
    1.
    2.
    3.

- Screenshot if needed:
![nosiytabset](https://cloud.githubusercontent.com/assets/17010094/25819715/bd6014da-3426-11e7-86e9-75b39f745589.gif)

- Any related issues:
cc: @bsclifton 
Please change milestone if required

**Comment by luixxiul** [comment]
Duping to https://github.com/brave/browser-laptop/issues/8292
